mne/decoding/classifier.py

```diff
@@ -395,7 +395,7 @@ def transform(self, epochs_data, y=None):
 
         if self.l_freq is None and self.h_freq is not None:
             epochs_data = \
-                low_pass_filter(epochs_data, self.fs, self.h_freq,
+                low_pass_filter(epochs_data, self.info['sfreq'], self.h_freq,
```

Comment:
I added the test. Any idea why only the first epoch gets filtered instead of all? The documentation for `low_pass_filter` says: "Applies a zero-phase lowpass filter to the signal x, operating on the last dimension.".